<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoadCalibParamsDialog</class>
 <widget class="QDialog" name="LoadCalibParamsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>348</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton_load">
   <property name="geometry">
    <rect>
     <x>191</x>
     <y>300</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Load</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_cancel">
   <property name="geometry">
    <rect>
     <x>283</x>
     <y>300</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>331</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Camera Left</string>
   </property>
   <widget class="QLineEdit" name="lineEdit_matrixL">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>200</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_matrixL">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>30</y>
      <width>90</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Matrix</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_distL">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>70</y>
      <width>90</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Distorsion</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_distL">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>200</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>331</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Camera Right</string>
   </property>
   <widget class="QLineEdit" name="lineEdit_matrixR">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>200</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_matrixR">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>30</y>
      <width>90</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Matrix</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_distR">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>70</y>
      <width>90</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Distorsion</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_distR">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>200</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
